src/components/Card/Card.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { render, cleanup, screen } from '@testing-library/react'
+import { shallow } from 'enzyme'
+import '@testing-library/jest-dom'
+import Card from './Card'
+
+const props = {
+  className: 'card1',
+  title: 'title',
+  alt: 'Video Tour | Welcome to Wizeline Guadalajara',
+  description: 'description',
+  imageLink: 'https://i.ytimg.com/vi/nmXMgqjQzls/mqdefault.jpg',
+  width: 200,
+  height: 90,
+}
+
+const imageProps = {
+  alt: 'title',
+  className: 'imageInside',
+  height: 90,
+  src: 'https://i.ytimg.com/vi/nmXMgqjQzls/mqdefault.jpg',
+  width: 200,
+}
+
+afterEach(cleanup)
+describe('testing Card component', () => {
```

Comment:
You could [write longer tests](https://kentcdodds.com/blog/write-fewer-longer-tests) instead of several short tests. No change required, just an observation.